spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -401,6 +401,25 @@ def __init__(self, parent=None, background_color=None):
         self.horizontalScrollBar().valueChanged.connect(
             self._set_hscrollbar_value
         )
+        self.register_shortcuts(parent.parent())
```

Comment:
```suggestion

        if parent is not None and parent.parent() is not None:
            self.register_shortcuts(parent.parent())
```

This should solve the errors in our tests